src/test/kotlin/com/terraformation/backend/customer/AppNotificationServiceTest.kt

```diff
@@ -634,11 +639,13 @@ internal class AppNotificationServiceTest : DatabaseTest(), RunsAsUser {
 
     insertUserDeliverableCategory(DeliverableCategory.GIS, user.userId)
     val deliverableId = insertDeliverable(deliverableCategoryId = DeliverableCategory.GIS)
+    val deliverable =
```

Comment:
I'm getting "No permission to read all deliverables" in these tests, but there is a     `every { user.canReadAllDeliverables() }` returns true in the setUp function, not sure what I'm doing wrong